### ISSUE TITLE
The BLEU4 score on SQuAD 1.1 zhou's split, finetune T5 base

### ISSUE THREAD

**Issue by zeaver** [salesforce/QGen#4]
Hi, thanks your open source code. I have fine tuned T5-base on SQuAD1.1 zhou's split.  The hyper-parameters: batch size: 32; warmp ratio: 0.1; learning rate: 1e-4, 7e-5, 5e-5 or 3e-5, 1e-5; beam num: 5. The BLEU4 score is on 20.0 ~ 21.2, never higher than 22.  And i found the result in your paper is 23.74. Could you please tell me your hyper-params? Thank you~
